assets/dir-index-html/src/dir-index.html

```diff
@@ -83,9 +83,12 @@
         </td>
         <td class="no-linebreak">
           {{ if .Hash }}
-          <a class="ipfs-hash" translate="no" href={{ if $root.DNSLink }}"https://cid.ipfs.io/#{{ .Hash | urlEscape}}" target="_blank" rel="noreferrer noopener"{{ else }}"{{ $root.GatewayURL }}/ipfs/{{ .Hash | urlEscape}}?filename={{ .Name | urlEscape }}"{{ end }}>
+          <a class="ipfs-hash" translate="no" href={{ if $root.DNSLink }}"https://cid.ipfs.io/#{{ .Hash | urlEscape}}" title="{{ .Hash }}" target="_blank" rel="noreferrer noopener"{{ else }}"{{ $root.GatewayURL }}/ipfs/{{ .Hash | urlEscape}}?filename={{ .Name | urlEscape }}"{{ end }}>
             {{ .ShortHash }}
           </a>
+          <button class="type-icon" title="Copy CID" onclick="copyText('{{ .Hash }}')">
```

Comment:
This code is slightly incorrect, if the CID was encoded with some fancy encoding that contains `'` it may break the string in the future.
You should encode the string in json to properly escape it :slightly_smiling_face:  (so I would do `copyText({{ .Hash | jsonEncode }})` or something like this).